### PR TITLE
Describe how this message is generated

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?
Pulling back the veil to the origin of this Chat message, describing where the action, and configuration live.

### CODE REVIEW

**Review by AshCorr — APPROVED** [review]
Good change!

File: src/main.rs [hunk 1]
```diff
@@ -148,6 +148,8 @@ async fn main() -> Result<(), Error> {
         message.push_str(format!("🧵 {} Reviews 🧵\n", weekday).as_str());
         message.push_str("(PR's can be hidden from this bot by adding the Stale tag)\n");
         message.push_str("--------------------\n\n");
+        message.push_str("This message is brought to you by guardian/actions-prnouncer, configured in guardian/prnouncer-config\n");
```
Comment (AshCorr) [inline src/main.rs:151]: Can we have links here? or is that a limitation of gchat?

Comment (akash1810) [reply src/main.rs]: Done (I assume no special markup is needed for GChat 🙏🏽 ).

**Comment by akash1810** [comment]
Hadn't anticipated GChat's rendering doing this! 

image

Not a huge fan of this...

**Comment by AshCorr** [comment]
Oops!

I think you could have used this syntax and that would stop it:

```
<{link url}|{link text}>
```
eg

```
<https://github.com/guardian/actions-prnouncer|guardian/prnouncer>
```

**Comment by akash1810** [comment]
> Oops!
> 
> I think you could have used this syntax and that would stop it:
> 
> ```
> <{link url}|{link text}>
> ```
> 
> eg
> 
> ```
> <https://github.com/guardian/actions-prnouncer|guardian/prnouncer>
> ```

👍🏽 I've raised https://github.com/guardian/actions-prnouncer/pull/12 and ran it locally this time to confirm behaviour 😅.

**Pull request merged**